src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,30 +1,56 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+type Props = {
+  query: string;
+  handleQueryChange: (value: string) => void;
+  handleSelectChange: (value: string) => void;
+  resetQuery: () => void;
+};
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+export const TodoFilter: React.FC<Props> = ({
+  query,
+  handleQueryChange = () => {},
+  handleSelectChange = () => {},
+  resetQuery = () => {},
```

Comment:
The default values for `handleQueryChange`, `handleSelectChange`, and `resetQuery` are unnecessary since these functions are always provided by the parent component. Consider removing these defaults to simplify the code.